code/game/objects/structures/displaycase.dm

```diff
@@ -106,13 +106,25 @@
 	return
 
 /obj/structure/displaycase/attackby(obj/item/weapon/W, mob/user, params)
-	if(W.GetID() && electronics && !broken)
+	if(W.GetID() && !broken)

```

Comment:
I think goof's right actually, the captain's case shouldn't be openable without breaking it. The RD's on the other hand is fine.